content/en-us/reference/engine/classes/Camera.yaml

```diff
@@ -534,15 +538,28 @@ properties:
     writeCapabilities: []
   - name: Camera.ViewportSize
     summary: |
-      Describes the dimensions, in pixels, of the client's viewport.
+      The dimensions of the device safe area on a Roblox client.
     description: |
-      **ViewportSize** describes the dimensions, in pixels, of the client's
-      viewport.
+      `Class.Camera.ViewportSize` returns the dimensions of the device safe area on the current screen. The device safe area is a rectangle which includes the Roblox top bar area, but does not include any cutouts in the display. The units of `Class.Camera.ViewportSize` are Roblox UI offset units, which may be different from native display pixels.
+
+      <img src="../../../assets/engine-api/enums/ScreenInsets/DeviceSafeInsets.png" width="840"
+      alt="Mobile device screen with cutout showing device safe area." />
+
+      Note: `Class.Camera.ViewportSize` is not equal to the fullscreen area size on displays with cutouts or notches. To obtain the fullscreen area size on all displays, you can query the `AbsoluteSize` property of a `Class.ScreenGui` with `ScreenInsets=None`. See `Enum.ScreenInsets` for a more information about how screen areas are defined.
+      `Class.Camera.ViewportSize` is not equal to the fullscreen size in order to improve cross-platform compatibility and backwards compatibility with legacy experiences.
+
+      <img src="../../../assets/engine-api/enums/ScreenInsets/None.png" width="840"
+      alt="Mobile device screen with cutout showing fullscreen area." />
+
+      Note: `Class.Camera.ViewportSize` is not the actual viewport size the camera uses for rendering. The Camera renders in the fullscreen area. Also the `Class.Camera.FieldOfView` and `Class.Camera.DiagonalFieldOfView` properties are based on the fullscreen area, not `Class.Camera.ViewportSize`.
+      
+      To learn more about how your experience will run on a device with notches/cutouts, please see [this post](https://devforum.roblox.com/t/notched-screen-support-full-release/2074324/1).
```

Comment:
```suggestion
```

Reply:
Best to not link to forum posts from API docs, since it can get confusing to understand based on all of the comments and back-and-forth discussion. If there's some key info in the post that should be added here, let's add it.